web/blog/2023-02-21-junior-dev-misconceptions.md

```diff
@@ -0,0 +1,122 @@
+
+<p align='center'>
+<figure>
+<img style={{ width: "100%" }} alt="Clean it up later" src="https://s3.us-west-2.amazonaws.com/secure.notion-static.com/f9a8b445-b5a0-42c6-9df1-14e3beba8a01/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20230222%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20230222T192813Z&X-Amz-Expires=86400&X-Amz-Signature=8a02b8c40b67783ee784f916ee1201eb1c1ccd2091004dab09ee667dd858a6e6&X-Amz-SignedHeaders=host&response-content-disposition=filename%3D%22Untitled.png%22&x-id=GetObject" />
+<figcaption class='image-caption' style={{ fontStyle: 'italic', opacity: 0.6, fontSize: '0.9rem'  }}>
+The most common misconception is "that you're going to come back and clean that up later." 
+</figcaption>
+</figure>
+</p>
+
+## Shipping Quality vs Shipping On Time
+
+The top reply above touches on all three of the common themes we outlined, because within it is a message about quality -- about doing things correctly. And whenever you speak about quality, there is an inherent assumption that it takes longer, so we're also talking about time management. And, if you're a part of a team, you can't work effectively without good communication and teamwork.
+
+Nevertheless, in the "quality" debate there were effectively two camps, with those who thought quality code was about: 
+ 1. writing clean, readable code, that's easy to maintain 
+ 2. writing code that gets shipped on time and works.
+
+The balance between meeting deadlines, shipping features, and writing the best possible code is obviously a tricky one to get right. Some people had the opinion that business realities trump clean code patterns in the dash to meet deadlines and keep clients happy, while others thought that clean, quality code should be the priority, and that by making it a priority you can actually increase long-term velocity, even if short-term deadlines aren't met.
+
+<p align='center'>
+<figure>
+<img style={{ width: "100%" }} alt="Clean it up later" src="https://s3.us-west-2.amazonaws.com/secure.notion-static.com/62b1a947-d8ef-406a-9b33-a89b30e23af3/Untitled.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Content-Sha256=UNSIGNED-PAYLOAD&X-Amz-Credential=AKIAT73L2G45EIPT3X45%2F20230222%2Fus-west-2%2Fs3%2Faws4_request&X-Amz-Date=20230222T194543Z&X-Amz-Expires=86400&X-Amz-Signature=ad7c767619af406e14e0183e0857eb7b8a75b94f2d07e95be7f998dd4ffed2d3&X-Amz-SignedHeaders=host&response-content-disposition=filename%3D%22Untitled.png%22&x-id=GetObject" />
+</figure>
+</p>
+
+This discussion can distract from Junior developers priorities though, which are to grow and improve as a developer, not lead the team to success. Therefore, it's probably best for Junior devs to focus on quality first, and then improve their speed of delivery second.
+
+## Staying Humble & Managing Expectations
```

Comment:
maybe make titles imperative, to sound more actionable: "Stay Humble & Manage Expectations". But that's just off the top of my head, feel free to disregard :D.

Reply:
Good idea. Changed all the titles to match